Extensions/people_notifier.js

```diff
@@ -360,6 +366,43 @@ XKit.extensions.people_notifier = new Object({
 			$(this).find('.count').css('marginRight', '0');
 		});
 
+		var dragging;
+		var dragtarget;
+		var mouseY;
+		$(document).on("dragstart", ".xkit-people-notifier-person", function(e){
+			dragging = $(this);
+			e.originalEvent.dataTransfer.dropEffect = "move";
+		});
+		$(document).on("drag", ".xkit-people-notifier-person", function(e){
+			mouseY = e.originalEvent.pageY;
+		});
+		$(document).on("dragenter", ".xkit-people-notifier-person", function(e){
+			e.preventDefault();
+			dragtarget = $(this);
+		});
+		$(document).on("dragover", ".xkit-people-notifier-person", function(e){
+			e.preventDefault();
+		});
+		$(document).on("drop", ".xkit-people-notifier-person", function(e){
+			if (dragtarget[0] !== dragging[0]) {
+				var target_index = XKit.extensions.people_notifier.blogs.indexOf(XKit.extensions.people_notifier.searchable_blogs[$(dragtarget).attr('data-url')]);
+				var dragging_obj = XKit.extensions.people_notifier.searchable_blogs[$(dragging).attr('data-url')];
+				var dragging_index = XKit.extensions.people_notifier.blogs.indexOf(dragging_obj);
+				dragging.detach();
+				if ((mouseY - dragtarget.offset().top) >= 12) {
```

Comment:
Since it's not immediately obvious to me that 12 is half the size of a div, we should store that in an appropriately named variable like `half_height_of_div` or something similarly descriptive.